### ISSUE TITLE
Benchmarking question

### ISSUE THREAD

**Issue by eds89** [carla-simulator/carla#463]
Hello,
Does the trained models (imitation learning, reinforcement learning, modular pipeline, etc) react to changes in the environment (input data)? For instance, if I extend an existing weather condition or create a post process, designed to add dust noise to the camera, will it impact on task benchmarking scores, when running it against existing trained data? I imagine that yes, but I'd like to double-check.

Kind regards,

**Comment by mvpcom** [comment]
@eds89 Generally, neural networks and machine learning models cannot generalize very well on entirely different and new sets. The train and test set should be from a similar distribution for an excellent performance. If you check the [results](http://carla.org/2018/04/23/release-0.8.2/), you can see for new town and weather, the performance of the model dramatically declined. So, you can expect extended a weather condition or created a post process will impact on task benchmarking scores.

**Comment by eds89** [comment]
Hi,
I wonder if the benchmarks are carried out with the actual camera data, as
the car navigates on the road, or with other game-provided data, e.g.
distance to obstacles, without making use of the captured image data? I'm
probably wrong on that one, but I'd like to double check.

**Comment by eds89** [comment]
Up!
